### ISSUE TITLE
Release 0.10.0

### ISSUE THREAD

**Issue by dblock** [slack-ruby/slack-ruby-client#170]

**Comment by jmanian** [comment]
In addition to the usual steps I should also change the version in [lib/slack/version.rb](https://github.com/slack-ruby/slack-ruby-client/blob/master/lib/slack/version.rb) (and in the changelog) to 0.10.0 *before* releasing, is that right?

**Comment by dblock** [comment]
Correct @jmanian

**Comment by jmanian** [comment]
I'll wait for #171 to merge then I'll rip it.

**Comment by dblock** [comment]
All merged.

**Comment by jmanian** [comment]
Ah shoot, I just realized I used yesterday's date in the changelog for the release (because I initially made the commit yesterday, then rebased after those PRs). This means the date in the changelog doesn't match the date on rubygems.

I think there are 4 options:
1. Leave it be
1. Update the date in the changelog in a new commit (which means [the date in v0.10.0](https://github.com/slack-ruby/slack-ruby-client/blob/v0.10.0/CHANGELOG.md) will always be wrong).
1. Amend the commit (which means the v0.10.0 tag won't identically match the published gem — they'll have different SHAs).
1. Amend the commit *and* republish the gem (the nuclear option).
